tests/cancun/eip6780_selfdestruct/test_dynamic_create2_selfdestruct_collision.py

```diff
@@ -39,7 +39,7 @@ def env():  # noqa: D103
     )
 
 
-@pytest.mark.valid_from("Paris")
+@pytest.mark.valid_from("Cancun")
```

Comment:
Note this is still in Draft. I changed these to Cancun for now such that these tests pass the validity marker check.

What do we do in cases like this? To me if a test is valid before the fork where the eip or feature is introduced, it should not exist within that fork folder.

Are they specific to eip6780 or selfdestruct? If its the latter, I think we should move them to Homestead (where selfdestruct was introduced, well renamed from `SUICIDE -> SELFDESTRUCT`). Then mark them valid from the forks as is currently without the change.